### DISCUSSION TITLE
1.5.1-beta3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#509] [Обновления]
## Список изменений
- HLS заменен на облегченную версию HLS light от @SashaXser в #506
- Убраны пути `/s/` и `/d/` из определения айди видео в Яндекс Диске (скрипт не запускался на этих путях, поэтому для конечного пользователя ничего не изменилось)
- Исправлена ошибка из-за которой значение автогромкости по умолчанию было 0.15%, а не 15%
- Исправлена ошибка из-за которой значение автогромкости могло иметь слишком много знаков после запятой

## What's Changed
* Облегченный hls.js by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/506

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1-beta2...1.5.1-beta3a

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1-beta3a'>1.5.1-beta3</a>.</em>

**Comment by ilyhalight** [comment]
убрал дублирование процента из хранилища

**Comment by NullVerdict** [comment]
Почему Яндекс иногда не может определить язык текста?

![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/98902aa6-b2aa-48b8-a6a3-9786d674b31b)

**Reply by ilyhalight** [reply]
Если текст состоит только из символов или цифр, тогда яндекс возвращает пустую строчку. Других вариантов, когда такое появляется, не встречал

**Comment by NullVerdict** [comment]
> Если текст состоит только из символов или цифр, тогда яндекс возвращает пустую строчку. Других вариантов, когда такое появляется, не встречал

Странно, можешь проверить логи таких ситуаций? Это вообще может произойти при любом символе

**Reply by ilyhalight** [reply]
Не логировались такие ситуации. Сейчас добавил логи и сразу заметил, что из-за наличия "http" (или "https") перестает возвращать язык, но иногда не возвращает из-за юникода (эмодзи, строки типо 𝒆𝒗𝒊𝒍), но в целом да, может и из-за любого слова перестать возвращать

**Reply by ilyhalight** [reply]
Немного собралось уже, закономерности какой-то если честно не увидел

https://pastebin.com/F4F0VRa9

**Comment by NullVerdict** [comment]
> Немного собралось уже, закономерности какой-то если честно не увидел
> 
> https://pastebin.com/F4F0VRa9

Определение не работает если: есть более 4 и более символов, эмоджи, юникод, из-за кривого удаления ссылки (вроде в 1.5.1 beta1 исправлено). Надо будет добавить в фильтр очистку лишних юникод символов, эмоджи и символов.

**Comment by elvin07137** [comment]
В udemy перевод не работает пишет ( не удалось получить данные, необходимые для перевода) уже месяц установил и старую и новую. Только что установил и 1.5.1 бету тоже не работает

**Reply by ilyhalight** [reply]
Обновите токен

**Reply by elvin07137** [reply]
обновил бро, даже кэш браузера. Ошибка опять

**Reply by elvin07137** [reply]
даже проверил в Edge  тоже выдает ошибку

**Reply by ilyhalight** [reply]
Значит на курсе нет видео в нужном формате, только, что проверил с курсом https://www.udemy.com/course/amazon-web-services-aws-v/ и там все работает

**Reply by elvin07137** [reply]
Я проверил больше 10 курсов не работает (платные, бесплатные), но который вы отправили работает. Такое ощущение что только в этом курсе перевод работает

**Comment by CrescentXIII** [comment]
Доброго! В Твиттере (Х) почему-то пишет "не найден ID" (в ВК так же частенько бывало. От чего это зависит? Спасибо.
